test/microsoft/sharepoint/FindFilesOrFolders.test.js

```
@@ -0,0 +1,470 @@
+            }
+        };
+
+        await action.receive(context);
+
+        const [data, outportName] = context.sendJson.args[0];
+
+        assert.equal(context.sendJson.callCount, 1, 'should call sendJson once');
+        assert.notDeepStrictEqual(data, {}, 'should NOT be an empty object');
+        assert.equal(outportName, 'out', "should be sent to 'out' output port");
+        assert.equal(data.count, totalFiles, `should return ${totalFiles} files in total`);
+        assert.equal(data.result[0].name.includes('testExcel'), true, `file 1 name should contain 'testExcel'`);
```

ESLint

[error] Strings must use singlequote.

(quotes)